Terminal-iOS/Study/SearchStudy/SearchStudyView.swift

```diff
@@ -0,0 +1,169 @@
+//
+//  SearchStudyViewController.swift
+//  Terminal-iOS
+//
+//  Created by once on 2020/09/02.
+//  Copyright © 2020 정재인. All rights reserved.
+//
+
+import UIKit
+import Then
+
+class SearchStudyView: UIViewController {
+    
+    var keyword: [String] = ["안드로이드ㅇㅇㅇㅇㅇㅇㅇ", "node.js", "코드리뷰", "취업스터디", "프로젝트", "Swift", "갓우석님", "예비유니콘케어닥fffff", "뭐지??왜안나옴"]
+        
+    let backBtn = UIButton()
+    let searchBar = UISearchBar()
+    let placeSearch = UIButton()
+    let hotLable = UILabel()
+    let tempView = UIView()
+    let collectionView: UICollectionView = {
+        let layout = LeftAlignedCollectionViewFlowLayout()
+        let view = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        return view
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = UIColor.appColor(.terminalBackground)
+        attribute()
+        layout()
+    }
+    
+    func attribute() {
+        backBtn.do {
+            $0.setImage(#imageLiteral(resourceName: "backButton"), for: .normal)
+            $0.addTarget(self, action: #selector(back), for: .touchUpInside)
+        }
+        searchBar.do {
+            $0.placeholder = "스터디명, 분류(키워드) 등"
+            $0.barTintColor = UIColor.appColor(.terminalBackground)
+        }
+        placeSearch.do {
+            $0.setTitle("장소로 검색", for: .normal)
+            $0.setTitleColor(.white, for: .normal)
+            $0.backgroundColor = UIColor.appColor(.mainColor)
+            $0.layer.cornerRadius = 10
+        }
+        hotLable.do {
+            $0.text = "핫 등록 키워드"
+            $0.textColor = .white
+            $0.dynamicFont(fontSize: 14, weight: .semibold)
+        }
+        collectionView.do {
+            $0.backgroundColor = UIColor.appColor(.terminalBackground)
+            $0.register(HotKeywordCell.self, forCellWithReuseIdentifier: HotKeywordCell.cellId)
+            $0.delegate = self
+            $0.dataSource = self
+        }
+        tempView.do {
+            $0.backgroundColor = UIColor.appColor(.terminalBackground)
+        }
+    }
+    
+    func layout() {
+        view.addSubview(backBtn)
+        view.addSubview(searchBar)
+        view.addSubview(placeSearch)
+        view.addSubview(hotLable)
+        view.addSubview(tempView)
+        view.addSubview(collectionView)
```

Comment:
```suggestion
        [backBtn, searchBar, placeSearch, hotLable, tempView, collectionView].forEach { view.addSubview($0) }
```

위 와 같은 방법도 좋을 것 같아 공유합니다.
추가로 tempView이름도 명시적으로 수정하면 좋을 것 같아요

Reply:
적용하겠습니다~